apps/shell/app.js

```diff
@@ -74,9 +72,14 @@ glob.sync(path.join((process.env.OOD_CLUSTERS || '/etc/ood/config/clusters.d'),
     if (isDefault) default_sshhost = host;
   });
 
+if (process.env.OOD_SSHHOST_ALLOWLIST) {
+  let parsed_allowlist = process.env.OOD_SSHHOST_ALLOWLIST.split(":")
+  host_allowlist = new Set(parsed_allowlist, ...host_allowlist);
+}
+
```

Comment:
As discussed offline, we want to revert the change here as that is unrelated to the support for wildcards.